dialogue-test-common/src/main/java/com/palantir/dialogue/AbstractChannelTest.java

```diff
@@ -97,6 +88,12 @@ public void close() {}
 
     protected Channel channel;
 
+    protected final MockWebServer server;
+
+    protected AbstractChannelTest(MockWebServer server) {
```

Comment:
I didn't see any tests out side this repo that extend this class, so while this is technically an API break I think we're fine